arch/arm/Kconfig

```diff
@@ -1279,6 +1279,17 @@ config ARM_FPU_ABI_SOFT
 	---help---
 		Pass float value via integer register (-mfloat-abi=softfp)
 
+config ARM_BUSY_WAIT
+	bool "Busy wait when boot"
+	default n
+	depends on ARCH_HAVE_MULTICPU
+	---help---
+		Enables busy wait when SMP boot
```

Comment:
Please explain more details in the help: why to use the busy wait in SMP boot, etc.